packages/web/src/components/Grid/_tools.scss

```diff
@@ -0,0 +1,34 @@
+@use 'sass:map';
+@use '@tokens' as tokens;
+@use '../../tools/breakpoint';
+
+@function breakpointClass($name, $value) {
+    @if ($value > 0) {
+        @return unquote($name + '--');
+    }
+
+    @return '';
+}
+
+@mixin gutters($gutters, $breakpoints) {
+    @each $breakpointName, $breakpointValue in $breakpoints {
+        @if not map.has-key($gutters, $breakpointName) {
+            @error 'Invalid breakpoint specified! #{$breakpointName} doesn\'t exist. Use one of #{map.keys($breakpoints)}';
+        }
+        @include breakpoint.up($breakpointValue) {
+            gap: map.get($gutters, $breakpointName);
+        }
+    }
+}
+
+@mixin basic-columns($grid-basic-columns, $breakpoints) {
+    @each $breakpointName, $breakpointValue in $breakpoints {
+        @include breakpoint.up($breakpointValue) {
+            @each $option in $grid-basic-columns {
+                .Grid--#{breakpointClass($breakpointName, $breakpointValue)}cols-#{$option} {
+                    grid-template-columns: repeat(#{$option}, 1fr);
```

Comment:
OK, let's start off with this 👍🏻 . We can enhance all these layouts to 12 columns with `> *` anytime later—if necessary.